lib/android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.airbnb.android.react.maps" >
+    <uses-sdk android:minSdkVersion="16" />
```

Comment:
Not sure if this is necessary, because build.gradle defines the minSdkVersion and would override the one defined in AndroidManifest.xml. Please correct me if I'm wrong.